commit 59afff8f19eee62e186669c24e9408e6917c7678
Author: Alexey Khoroshilov <khoroshilov@ispras.ru>
Date:   Sun Mar 5 03:01:55 2017 +0300

    net/sched: act_skbmod: remove unneeded rcu_read_unlock in tcf_skbmod_dump
    
    [ Upstream commit 6c4dc75c251721f517e9daeb5370ea606b5b35ce ]
    
    Found by Linux Driver Verification project (linuxtesting.org).
    
    Signed-off-by: Alexey Khoroshilov <khoroshilov@ispras.ru>
    Acked-by: Jamal Hadi Salim <jhs@mojatatu.com>
    Signed-off-by: David S. Miller <davem@davemloft.net>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1394_59afff8f19eee62e186669c24e9408e6917c7678.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1394_59afff8f19eee62e186669c24e9408e6917c7678.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
